src/Stryker.Core/Stryker.Core/Mutators/ExclusiveOrMutator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+using System.Collections.Generic;
+
+namespace Stryker.Core.Mutators
+{
+    public class ExclusiveOrMutator : MutatorBase<BinaryExpressionSyntax>, IMutator
```

Comment:
My idea with this class was to keep the BinaryExpressionMutator class simple since the method used to transform expressions there would not work in this case. But I see how keeping these in one class is less surprising.